tck-runner/src/test/java/org/hibernate/validator/tckrunner/securitymanager/arquillian/LocalSecurityManagerTestingExecutionEvent.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.tckrunner.securitymanager.arquillian;
+
+import java.lang.reflect.Method;
+
+import org.hibernate.validator.tckrunner.securitymanager.DelegatingExecutor;
+import org.hibernate.validator.tckrunner.securitymanager.Executor;
+import org.jboss.arquillian.container.test.impl.execution.event.LocalExecutionEvent;
+import org.jboss.arquillian.test.spi.TestMethodExecutor;
+
+/**
+ * A custom {@link LocalExecutionEvent} which channels execution through a specific protection domain.
+ *
+ * @author Gunnar Morling
+ */
+public class LocalSecurityManagerTestingExecutionEvent extends LocalExecutionEvent {
+
+	public LocalSecurityManagerTestingExecutionEvent(TestMethodExecutor executor) {
+		super( new DelegatingTestMethodExecutor( executor ) );
+	}
+
+	/**
+	 * Invokes the given test method via {@link DelegatingExecutor} which lives in its own protection domain.
+	 *
+	 * @author Gunnar Morling
```

Comment:
It's part of my IDE templates for new classes and I didn't really care about it. Feel free to remove it if you like.